### DISCUSSION TITLE
How do you specify a path for typescript?

### DISCUSSION THREAD

**Discussion by PomegranateApps** [callstack/react-native-builder-bob#297] [Q&A]
How do you specify a path for tsc? 

✖ The tsc binary doesn't seem to be installed under node_modules or present in $PATH. Make sure you have added typescript to your devDependencies or specify the tsc option for typescript.

**Comment by satya164** [comment]
If it's in your `devDependencies` then it'll be auto-detected. If it's not working, you can provide a repro project to check.
